scalableminds/webknossos
# Color Logging for local development

Adding a second logback.xml for local development only that re-enables color logging (highlights warnings/errors)

### URL of deployed dev instance (used for testing):
- https://colorloggingdev.webknossos.xyz

### Steps to test:
- Run locally via proxy.js (yarn start)
- backend logging should highlight warnings/errors with color
- open dev deployment logging in text editor like sublime
- should *not* include ansii color characters

### Issues:
- fixes #3549 

------
